packages/create-burner/test/utils/storage.test.ts

```diff
@@ -0,0 +1,26 @@
+import { describe, it, expect, vi } from "vitest";
+import Storage from "../../src/utils/storage";
+import Cookies from "js-cookie";
+
+vi.mock("js-cookie");
+
+describe("storage", () => {
+    it("should return null", async () => {
+        Cookies.get = vi.fn().mockReturnValue({});
+        expect(Storage.keys()).toStrictEqual([]);
+    }),
+        it("should return a json", async () => {
+            const jsonObj = '{ "myString": "string", "myNumber": 4 }';
```

Comment:
could you match this to the burners type object rather than a default json

Reply:
Done. Added a BurnerStorage type object